### PR TITLE
MTL-1858 Fix Tests and Consolidate Roles 

### PR DESCRIPTION
**Pull request by rustydb**
### Summary and Scope

- Fixes:  MTL-1858

#### Issue Type

- RFE Pull Request

All tests for NCNs are ran in the final images to mititage issues with enabled/running and true/false.

This merges `ncn-common-setup` and `ncn-common-install` together, the `node-images` pipeline will invoke the new role `ncn-common` after all RPMs have been installed.

This adds an `ansible.cfg`.

The `ncn-common-team` role was renamed to the team it belongs too.

The `default/main.yml` files were broken up into new `var/{common,google,metal}.yml` files that are loaded based on context. This simplified several of the tasks referring to Daemons.

Many `copy` tasks were replaced with `synchronize` because I was going bonkers trying to merge the logic of loading `files/` into `/srv/cray` between the packer pipeline and this repo.

### Prerequisites

- [ ] I have included documentation in my PR (or it is not required)
- [x] I have tested this by building an image using this branch HEAD, or this is does not pertain to the node-images pipeline.
- [ ] I tested this on internal system or this pertains to the node-images pipeline (if yes and not build related, please include results or a description of the test)
- [ ] I tested this on a vshasta system or this pertains to the node-images pipeline (if yes and not build related, please include results or a description of the test)

### CODE REVIEW

File: ansible.cfg
Comment (jpdavis-prof) [inline ansible.cfg:3]: Can we take **ANSIBLE_FORCE_COLOR=1 PYTHONUNBUFFERED=1 ANSIBLE_STDOUT_CALLBACK=debug**
out of the packer config and add the equivalent here too?
Definitely no reason to hold up merging for it though.

Comment (jpdavis-prof) [inline ansible.cfg:3]: Can we take ANSIBLE_FORCE_COLOR=1 PYTHONUNBUFFERED=1 ANSIBLE_STDOUT_CALLBACK=debug
out of the packer config and add the equivalent here too?
Definitely no reason to hold up merging for it though.

**Review by jpdavis-prof — APPROVED** [review]

File: ansible.cfg
Comment (rustydb) [reply ansible.cfg:3]: Think so. I do want to rebuild after merging this to main.

Comment (rustydb) [reply ansible.cfg:3]: I also wanted to enable color for packer, which is disabled in the CSM shared lib. Maybe we can make a follow up Jira for those two items (ansible.cfg and csm-shared-library).

Comment (rustydb) [reply ansible.cfg:3]: I'll just make the changes with this Jira. Lazy.

Comment (rustydb) [reply ansible.cfg:4]: I'll add ANSIBLE_FORCE_COLOR but I'll leave the other two, I can't move PYTHONUNBUFFERED but I also don't want to change the runtime Ansible too much.

**Pull request merged**
